### PR TITLE
Adding Simulated Tracker Hits

### PR DESCRIPTION
**Pull request by cbravo135**
Simulated tracker hits are added as MCTrackerHit in event. A corresponding processor is also added.

### CODE REVIEW

File: event/include/Collections.h [hunk 1]
```diff
@@ -46,6 +46,9 @@ namespace Collections {
     /** Name of the collection containing MC Particles. */
     constexpr const char* MC_PARTICLES{"MCParticle"}; 
 
+    /** Name of the collection containing MC Particles. */
```
Comment (pbutti) [inline event/include/Collections.h]: Comment wrong?

File: event/include/MCTrackerHit.h [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        /** The hit time. */
+        double time_{-999};
+
+        /** Layer (Axial + Stereo). 1-12 in 2015/2016 geometry, 1-14 in 2019 geometry */
+        int layer_{-999};
+
+        /** Module */
+        int module_{-999};
+
+        /** Raw charge: sum of the raw hit fit amplitudes */
```
Comment (pbutti) [inline event/include/MCTrackerHit.h]: Wrong comment? How the edep can be the raw hit fit?

**Review by pbutti — COMMENTED** [review]
I think this is ok.
For improved readability, clean up those couple of comments

File: event/include/Collections.h (hunk 1 above)
Comment (cbravo135) [reply event/include/Collections.h]: fixed

File: event/include/MCTrackerHit.h (hunk 2 above)
Comment (cbravo135) [reply event/include/MCTrackerHit.h]: fixed

**Review by pbutti — APPROVED** [review]

**Pull request merged**